### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by MartinBasti**
Dependabot is GitHub tool for automated updates of released
dependencies and CVE fixes.

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/about-dependabot-version-updates

Signed-off-by: Martin Basti <mbasti@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request has a link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by ben-alkov — APPROVED** [review]
🚀

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
```
Comment (chmeliik) [inline .github/dependabot.yml:10]: empty line

**Review by chmeliik — APPROVED** [review]
LGTM, just a cosmetic flaw

**Comment by chmeliik** [comment]
oh, and "dependabot", not "dependant" bot

**Comment by MartinBasti** [comment]
> oh, and "dependabot", not "dependant" bot

Yeah I cannot read/write, fixed

**Comment by MartinBasti** [comment]
no release notes, only Github tooling change

**Pull request merged**